.github/workflows/shellcheck.yml

```diff
@@ -34,15 +36,13 @@ jobs:
             github.com:443
             pkg-containers.githubusercontent.com:443
       - uses: actions/checkout@v2
-      - name: Force Install latest shellcheck
-        run: |
-          eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)" && \
-          brew install shellcheck --prefix=/usr/local/bin/
+      - name: Install latest shellcheck
+        run: $brew_bin/brew install shellcheck
         env:
           HOMEBREW_NO_ANALYTICS: 1
-      - run: "shellcheck --version"
+      - run: "$brew_bin/shellcheck --version"
       - name: Run shellcheck on ${{ matrix.file }}
-        run: shellcheck -s ${{ matrix.shell }} ${{ matrix.file }}
+        run: $brew_bin/shellcheck -s ${{ matrix.shell }} ${{ matrix.file }}
```

Comment:
oof :-/ can we add `$brew_bin` to the PATH, so that the manual path prefixes aren't required?

Reply:
I was actually considering going the other way and just referencing the full path instead of a env var so its clear in the runner logs which shellcheck was used.  Currently `which shellcheck` and `command -v shellcheck` return 2 different shellchecks in the runner, ubuntu packaged shellcheck and brew install shellcheck.   I guess it doesn't really matter which is used, as long as the `shellcheck --version` returns the current version.  However, considering we go to the trouble of making sure to install the latest version, I would like to be sure that instance was used.

Reply:
After adding to the PATH, it'd be fine to output a `which shellcheck` so we know which one's being used :-)

Reply:
I added a `which shellcheck` call in commit https://github.com/nvm-sh/nvm/pull/2907/commits/1adc3432ca39fa9625a79fd5150a00727c858308